src/pybind/mgr/dashboard/frontend/src/app/core/auth/login/login.component.scss

```diff
@@ -1,31 +1,21 @@
-@import '../../../../defaults';
-
-.login {
-  height: 100%;
-
-  .row {
-    color: $color-login-row-text;
-    background-color: $color-login-row-bg;
-  }
-
-  h1 {
-    margin-top: 0;
-    margin-bottom: 30px;
-  }
+.branding-border {
+  width: calc(100% + 60px);
```

Comment:
What are these two first one for (`width: 100% + 60px` and `margin-left: -30px`)?

Reply:
@epuertat RCUE branding doesn't seem to provide a top red border for login view. Since login div is child view of app component, I had to come up with this work-around. I added a div and styled it as the border. The width and margin are styling the same div.

Reply:
I've only seen the red stripe at login screen in that screenshot provided by @julienlim, but [RCUE does not show that element](https://brand.redhat.com/applications/product-ux/). In Patternfly it exists (blueish) but is part of the login screen [background image](https://www.patternfly.org/components/patternfly/dist/img/bg-login.jpg). [Other screenshots of Red Hat products' login screens](https://www.google.com/search?q=red+hat+login&tbm=isch) don't show that red stripe either. My point is to just keep the branding as standard and straightforward as possible, given the upstream is not patternfly based. If that red stripe is not mandatory, I'd avoid it, as long as adding that on our own will require us to check that with different screen sizes and devices.

Reply:
@a2batic @epuertat Regarding the red strip at the top of the login screen, I just spoke with Kyle Baker, who tells me the red stripe is optional now.  As it's no longer mandatory, let's leave it off to make the branding effort easier.

Just curious, is there any particular reason why we have extra hint words in the input text boxes?  Specifically, do we really need to be verbose and say "Enter your username" (vs. "Username") and "Enter your password" (vs. "Password")?  Is this due to downstream?

Otherwise, everything looks good!

Reply:
@julienlim Thanks for the clarification! @a2batic has an exceptional eye for the detail (I completely overlooked that strip/stripe). The thing is that we still have the upstream branding recommendations by Michael pending to do, and that's gonna change completely the login screen layout, so perhaps that's a good chance for improving this.

Reply:
@julienlim @epuertat ack, I will remove the top red border and "Enter your" phrase.